packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -471,205 +403,161 @@ Cleaning the wishlist can be achieved by `clear` property.
-        isInWishlist,
-        loading
-      };
-    }
-  };
+  <ul>
+    <li v-for="product in products" :key="productGetters.getId(product)">
+      <button @click="addToCart({ product, quantity })">
+        Add to cart
+      </button>
+      <button @click="addToWishlist({ product })">
+        Add to wishlist
+      </button>
+      <span v-if="isInCart({ product })">
+        added to cart
+      </span>
+      <span v-if="isInWishlist({ product })">
+        added to wishlist
+      </span>
+    </li>
+  </ul>
+</template>
+<script>
+import { useCart, useWishlist, useFacet, facetGetters } from '{INTEGRATION}';
+
+export default {
+  setup() {
+    const { addItem: addToCart, isInCart } = useCart();
+    const { addItem: addToWishlist, isInWishlist } = useWishlist();
+    const { result } = useFacet();
+    
+    const products = computed(() => facetGetters.getProducts(result.value));
+
+    const quantity = ref(1);
+
+    return {
+      products,
+      addToCart,
+      isInCart,
+      addToWishlist,
+      isInWishlist,
+      quantity
+    };
+  }
+};
 </script>
 ```
 
 The cart component:
 
 ```vue
 <template>
-    <div>
-      <ul>
-        <li
-          v-for="product in cartProducts" :key="product.id"
-        > 
-          <input 
-            type="number" 
-            v-model="quantity"
-          >
-          <button
-            @input="updateItemQty({ product, quantity })"
-          >
-            Change quantity
-          </button>
-          <button        
-            @click="removeItem({ product })"
-          >
-            Remove from cart
-          </button>      
-        </li>
-      </ul>
-      <span>
-        {{ cartTotals.total }}
-      </span>
-      <span>
-        {{ cartTotalItems }}
-      </span>
-      <button
-        @click="clear"
-      >
-        Clear cart
-      </button>
-    </div>
-</template> 
-<script> 
-  import { computed, ref } from '@vue/composition-api';
-  import { useCart, cartGetters } from '{INTEGRATION}';
-  import { onSSR } from '@vue-storefront/core';
-  export default {
-    setup() {
-      const {
-        cart,
-        clear,
-        removeItem, 
-        updateItemQty,
-        load,
-      } = useCart();
-
-      const cartProducts = computed(() => cartGetters.getItems(cart.value));
-      const cartTotals = computed(() => cartGetters.getTotals(cart.value));
-      const cartTotalItems = computed(() => cartGetters.getTotalItems(cart.value));     
-
-      const quantity = ref(0);
-
-      onSSR(async () => {
-        await load();
-      });
-
-      return {
-        cartProducts,
-        cartTotals,
-        cartTotalItems,
-        removeItem,
-        cart,
-        updateItemQty,
-        clear, 
-        quantity
-      };
-    },
-  };
+  <div>
+    <ul>
+      <li v-for="item in cartItems" :key="cartGetters.getItemSku(product)">
+        <input type="number" v-model="quantity" />
+        <button @click="updateItemQty({ item, quantity })">
```

Comment:
The same issue as mentioned above - we loop through products, but only have a single `quantity`